photon-client/src/lib/PhotonUtils.ts

```diff
@@ -18,3 +20,32 @@ export const parseJsonFile = async <T extends Record<string, any>>(file: File):
     fileReader.readAsText(file);
   });
 };
+
+export const axiosPost = (url: string, data?: any, config?: any): Promise<void> => {
+  return axios
+    .post(url, data, config)
+    .then(() => {
+      useStateStore().showSnackbarMessage({
+        message: "Successfully dispatched the command. Waiting for backend to respond",
```

Comment:
This command feels pretty generic for our users. Can we make this more helpful? Ditto on other references to requests.